internal/pkg/manifest/svc.go

```diff
@@ -647,10 +742,18 @@ func (s *BackendService) MainContainerPort() string {
 	return port
 }
 
-func prepareParsedExposedPortsMap(exposedPorts []ExposedPort) (map[string][]ExposedPort, map[uint16]string) {
+func prepareParsedExposedPortsMap(exposedPorts map[uint16]ExposedPort) (map[string][]ExposedPort, map[uint16]string) {
 	parsedContainerMap := make(map[string][]ExposedPort)
 	parsedExposedPortMap := make(map[uint16]string)
-	for _, exposedPort := range exposedPorts {
+
+	ports := []uint16{}
+	for port := range exposedPorts {
+		ports = append(ports, port)
+	}
+	sort.Slice(ports, func(i, j int) bool { return ports[i] < ports[j] })
```

Comment:
Why do we need to sort the ports?

Reply:
Just to make it easier to do unit tests for.